app/models/need.rb

```diff
@@ -173,7 +175,7 @@ def self.dynamic_fields
   end
 
   def self.categories_for_triage
-    categories.except('Other')
+    categories.except('Other').reject { |_k, v| v.in? ASSESSMENT_CATEGORIES }
```

Comment:
Little trick: you could do `categories.except('Other', *ASSESSMENT_CATEGORIES)` (it will expand the array to be arguments)